os/services/rpl-border-router/native/Makefile.native

```diff
@@ -4,10 +4,16 @@ MODULES += os/services/shell
 MAKE_MAC = MAKE_MAC_OTHER
 MAKE_NET = MAKE_NET_IPV6
 
-
 PREFIX ?= fd00::1/64
+
+ifneq ($(PORT),)
+	ROUTER_ARGS = -s $(PORT) $(PREFIX)
+else
+	ROUTER_ARGS = $(PREFIX)
```

Comment:
Does
```Makefile
ifneq ($(PORT),)
    ROUTER_ARGS = -s $(PORT)
endif
ROUTER_ARGS += $(PREFIX)
```
accomplish the same thing but without the else-branch?

Reply:
Agreed. Updated.